### DISCUSSION TITLE
Contact a user about a shared project interest?

### DISCUSSION THREAD

**Discussion by bradshawlupton** [community/community#37483] [New to GitHub]
I would like to contact **KGx55** about his wwv chu GQRX sever monitoring tools and experiences.

I wish to do the same then similar monitoring at my Sandwich STEM Academy and Massachusetts Maritime Academy research  projects at either end of the cape cod canal an hour south of Boston.

is there a facility for user-user inter communication?

thanks, bradshaw

[REDACTED DUE TO PII]
**K1te**

**Comment by queenofcorgis** [comment]
Hi @bradshawlupton ,

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md). The best way to contact this user is by opening an Issue in the repo for that project. We do not offer private messaging at this time. Best of luck!

**Comment by macbugs** [comment]
That user has no public repositories and is not the maintainer of this project -- he merely [opened an issue](https://github.com/gqrx-sdr/gqrx/issues/1128) in [github.com/gqrx-sdr/gqrx](https://github.com/gqrx-sdr/gqrx), which provides a link to the [project web site](https://gqrx.dk/) (which contains some [documentation](https://gqrx.dk/category/doc) and a link to the [official support forum](https://gqrx.dk/user-support).)  If you really need to contact that specific user, replying to his issue would be the only way, but the official support forum would probably be your best option if you have questions regarding applications.

**Reply by bradshawlupton** [reply]
Thanks for the insight and help.

kgx55 is doing things I wish to do.

I will pursue the route you suggested.         TNX. I also appreciate your redaction!

 If you really need to contact that specific user, replying to his issue would be the only way, but the official support forum would probably be your best option if you have questions regarding applications.
